stories/small_multiples/4_vertical_bars.tsx

```diff
@@ -0,0 +1,165 @@
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs';
+import { DateTime } from 'luxon';
+import React from 'react';
+
+import {
+  ScaleType,
+  Position,
+  Chart,
+  Axis,
+  GroupBy,
+  SmallMultiples,
+  Settings,
+  BarSeries,
+  LineAnnotation,
+  AnnotationDomainTypes,
+  LIGHT_THEME,
+  LineSeries,
+} from '../../src';
+import { SeededDataGenerator } from '../../src/mocks/utils';
+import { ColorVariant } from '../../src/utils/common';
+import { SB_SOURCE_PANEL } from '../utils/storybook';
+
+const dg = new SeededDataGenerator();
+const numOfDays = 7;
+function generateData(totalGroups = 2, mappings: Record<string, string>) {
+  return dg.generateGroupedSeries(numOfDays, totalGroups).map((d) => {
+    return {
+      ...d,
+      x: DateTime.fromFormat(`${d.x + 1}`, 'E').toFormat('EEEE'),
+      y: Math.floor(d.y * 10),
+      g: mappings[d.g],
+    };
+  });
+}
+const data1 = generateData(2, { a: 'new user', b: 'existing user' }).map((d) => ({ ...d, site: 'website A' }));
+const data2 = generateData(2, { a: 'new user', b: 'existing user' }).map((d) => ({ ...d, site: 'website B' }));
+
+const datal1 = generateData(2, { a: 'avg new user', b: 'avg existing user' }).map((d) => ({
+  ...d,
+  site: 'website A',
+}));
+const datal2 = generateData(2, { a: 'avg new user', b: 'avg existing user' }).map((d) => ({
+  ...d,
+  site: 'website B',
+}));
+
+export const Example = () => {
+  const marker = (
+    <span
+      style={{
+        backgroundColor: 'lightgray',
+        padding: 2,
+        width: 30,
+        height: 10,
+        margin: 'auto',
+        fontSize: 8,
+        borderRadius: 2,
+      }}
+    >
+      MIN
+    </span>
+  );
+  const showLegend = boolean('Show Legend', true);
+  const onElementClick = action('onElementClick');
+  const disableSmallMultiples = boolean('Disable small multiples', false);
+
+  return (
+    <Chart className="story-chart">
+      <Settings onElementClick={onElementClick} rotation={0} showLegend={showLegend} />
+      <Axis id="time" title="metric" position={Position.Bottom} gridLine={{ visible: false }} />
+      <Axis id="y" title="Day of week" position={Position.Left} gridLine={{ visible: false }} />
```

Comment:
Looks like the titles are flipped

```suggestion
      <Axis id="time" title="Day of week" position={Position.Bottom} gridLine={{ visible: false }} />
      <Axis id="y" title="Metric" position={Position.Left} gridLine={{ visible: false }} />
```

![image](https://user-images.githubusercontent.com/19007109/107535905-cc299900-6b86-11eb-9bbc-4cdadf4b614e.png)

Reply:
done https://github.com/elastic/elastic-charts/pull/999/commits/93b41b46b6f3cd01ec3408c87ee4f6bb4fd9304a